main.py

```diff
@@ -0,0 +1,78 @@
+import cv2
```

Comment:
What is the point of copying all of this into a .py file? It's more useful in the notebook for testing anyways.

Reply:
It runs faster in a python file